packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -497,19 +497,23 @@ mixin WidgetsBinding
   ///
   /// - If true, taps in the app are intercepted by the widget inspector.
   /// - If false, taps in the app are not intercepted by the widget inspector.
-  ValueNotifier<bool> get debugWidgetInspectorSelectionOnTapEnabledNotifier =>
-      _debugWidgetInspectorSelectionOnTapEnabledNotifierObject ??= ValueNotifier<bool>(true);
-  ValueNotifier<bool>? _debugWidgetInspectorSelectionOnTapEnabledNotifierObject;
+  ValueListenable<bool> get debugWidgetInspectorSelectionOnTapEnabledListenable =>
+      _debugWidgetInspectorselectionOnTapEnabledNotifier;
+  final ValueNotifier<bool> _debugWidgetInspectorselectionOnTapEnabledNotifier =
+      ValueNotifier<bool>(true);
+
+  set debugWidgetInspectorSelectionOnTap(bool value) {
+    _debugWidgetInspectorselectionOnTapEnabledNotifier.value = value;
+  }
```

Comment:
it we are exposing a setter to set the notifier value, then let's just expose the notifier directly rather than having getters and setters that are essentially wrapping the same functionality. And then we can delete the getter and setter:

```dart
final ValueNotifier<bool> debugWidgetInspectorSelectionOnTapEnabled =
      ValueNotifier<bool>(true);
```

Reply:
done!